dpnp/fft/dpnp_iface_fft.py

```diff
@@ -36,18 +36,8 @@
 
 """
 
-# pylint: disable=invalid-name
-
-import numpy
-
 import dpnp
 
-# pylint: disable=no-name-in-module
-from dpnp.dpnp_utils import (
-    call_origin,
-    checker_throw_axis_error,
```

Comment:
I believe we can clean up and to remove `normalize_axis` and `checker_throw_axis_error` from backend code